### PR TITLE
[xcvrd] Ignore processing ports that are not front-panel

### PR DESCRIPTION
**Pull request by vivekrnv**
#### Description

Ignore processing the front panel ports in xcvrd. This include ports with suffixes other than Ethernet<num> and ports that have role field set to any value other than Ext.

### CODE REVIEW

File: sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py [hunk 1]
```diff
@@ -96,6 +109,16 @@ def subscribe_port_update_event(self):
                                             port_tbl, list(d.values())[0], namespace))
         self.sel, self.asic_context = sel, asic_context
 
+    def is_valid_physical_port(self, port_name):
```
Comment (oleksandrivantsiv) [inline sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py]: I don't know if the name of the function is accurate. Why do we use `is_front_panel_port` instead?

Comment (vivekrnv) [reply sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py]: I actually don't need this method at all. I can just inline the actual call

Comment (vivekrnv) [reply sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py]: Updated

**Review by oleksandrivantsiv — APPROVED** [review]

File: sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py [hunk 2]
```diff
@@ -119,7 +130,7 @@ def handle_port_update_event(self):
                     (port_name, op, fvp) = port_tbl.pop()
                     if not port_name:
                         break
-                    if not validate_port(port_name):
+                    if not multi_asic.is_front_panel_port(port_name, self.port_role_map.get(port_name, None)):
```
Comment (dgsudharsan) [inline sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py]: How will this logic work for dynamic ports? If there is DPB I don't think port_role_map gets updated

**Review by dgsudharsan — CHANGES_REQUESTED** [review]

File: sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py (hunk 2 above)
Comment (vivekrnv) [reply sonic-xcvrd/xcvrd/xcvrd_utilities/port_event_helper.py]: Good catch!, Updated the logic to handle DPB

**Pull request closed without merging**